### DISCUSSION TITLE
Use x:DataType in Xaml file

### DISCUSSION THREAD

**Discussion by aahmyu** [MvvmCross/MvvmCross#4128] [Q&A]
Hello,

How can I fix these question marks when using mvvmCross? I'm using jetbrains rider.

![image](https://user-images.githubusercontent.com/5700557/111217035-61300f80-85d5-11eb-9105-ea7677558b53.png)

Normally I can specify a x:DataType or a binding context for an element and the path will show normally.
`x:DataType="viewModel:SideNavViewModel"`

![image](https://user-images.githubusercontent.com/5700557/111218438-05668600-85d7-11eb-9a63-4d9158368480.png)

What is happening here is the path works correctly on the `MenuItemList` property but It doesn't for the list items. If I remove the `DataType` the items text works.

Edit: 

I added a DataType to the DataTemplate and it works

![image](https://user-images.githubusercontent.com/5700557/111222962-c9ceba80-85dc-11eb-9b8d-161fad56e723.png)

Is this really the right way to do it? 

Regards

**Comment by Cheesebaron** [comment]
I don't see how this is related to MvvmCross. Open an issue with the Rider team.

**Comment by Hackmodford** [comment]
You could also use the markup for C# extensions from the Xamarin Community Toolkit  as an alternative.
https://docs.microsoft.com/en-us/xamarin/community-toolkit/markup
